### PR TITLE
Feat/calendar style

### PR DESCRIPTION
**Pull request by oleksiuk-ol**
image

https://trello.com/c/1P5Y6yXT/185-fe-fix-calendar-style

### CODE REVIEW

File: src/components/calendar/Calendar.sass [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.fc .fc-toolbar-title
```
Comment (nsukhotskaya) [inline src/components/calendar/Calendar.sass:1]: fc - Do you think this is a good naming ?

File: src/components/calendar/Calendar.sass [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    transition: .6s cubic-bezier(.25,.8,.25,1)
+
+.fc .fc-button-primary:hover
+    background-color: #005b8d
+    border-color: #005b8d
+
+.fc .fc-button-primary:not(:disabled).fc-button-active
+    background-color: #004b74
+    border-color: #004b74
+
```
Comment (nsukhotskaya) [inline src/components/calendar/Calendar.sass:20]: 3 diffrents background colors/ Please attach screen with your design

File: src/components/calendar/Calendar.sass (hunk 1 above)
Comment (oleksiuk-ol) [reply src/components/calendar/Calendar.sass:1]: By fullcalendar docs I need to override properties

File: src/components/calendar/Calendar.sass (hunk 2 above)
Comment (oleksiuk-ol) [reply src/components/calendar/Calendar.sass:20]: I suppose? For hover, active button and by default

**Review by nsukhotskaya — APPROVED** [review]

**Review by OstroumovAlex — APPROVED** [review]

**Pull request merged**